packages/vm/src/edge-vm.ts

```diff
@@ -89,6 +90,7 @@ function addPrimitives(context: VMContext) {
   defineProperty(context, 'clearTimeout', { value: clearTimeout })
   defineProperty(context, 'setInterval', { value: setInterval })
   defineProperty(context, 'setTimeout', { value: setTimeout })
+  defineProperty(context, 'EdgeRuntime', { value: 'edge-runtime' })
```

Comment:
**suggestion:** maybe we could make it non-writable?

By default `defineProperty` creates writable properties (despite not being configurable, their value could still be [changed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#configurable)).

Reply:
I think it's fine for it to be writable. Maybe someone would want to extend it with a custom flavor and set it to `custom-edge-runtime`. what do you think?

Reply:
well, it's JS. We should not be too prescriptive! 😅 
Let's go with writable.